skyportal/handlers/api/source.py

```diff
@@ -491,6 +492,18 @@ def get(self, obj_id=None):
             description: |
               Comma-separated string of "taxonomy: classification" pair(s) to filter for sources matching
               that/those classification(s), i.e. "Sitewide Taxonomy: Type II, Sitewide Taxonomy: AGN"
+          - in: query
+            name: annotationsFilter
+            nullable: true
+            schema:
+              type: array
+              items:
+                type: string
+            explode: false
+            style: simple
+            description: |
+              Comma-separated string of "annotation: value: operator" pair(s) to filter for sources matching
+              that/those annotation(s), i.e. "redshift: 0.5: lt"
           - in: query
```

Comment:
Add `annotationsFilterOrigin` that will only accept specific origin(s) for filtering with `annotationsFilter`. 
(assume some rogue origin is posting unrealistic results that you want to avoid)

Reply:
added.